exec/vm.go

```diff
@@ -60,6 +60,13 @@ type VM struct {
 	compiledFuncs []compiledFunction
 
 	funcTable [256]func()
+
+	// RecoverExecutionPanic controls whether the `ExecCode` method
+	// recovers from execution panics and returns them as errors
```

Comment:
The name and comment mention "execution panic", but doesn't really define it.

What is an "execution panic" and how does it differ from a normal Go run-time panic (as defined at https://golang.org/ref/spec#Handling_panics and https://golang.org/ref/spec#Run_time_panics)?

If by "execution panic" you simply refer to normal Go panics that happen while executing a VM, then I would suggest getting rid of "execution panic" and make it just:

```Go
// RecoverPanic controls whether the `ExecCode` method recovers from panics
// during execution of the VM and returns them as errors instead.
// A panic can occur either when executing an invalid VM
// or encountering an invalid instruction, e.g. `unreachable`.
RecoverPanic bool
```

I think that would be more clear and understandable for readers of the documentation.

If "execution panic" refers to something else defined somewhere else, then it'd be great to reference it, because you can see how it might be misinterpreted otherwise.